app/controllers/admin/locations_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::LocationsController < AdminController
+  def index
+    @locations = Location.all
```

Comment:
Does calling .order without .all do the same thing?
I'm not sure, can you check?

Reply:
Can confirm works the same. So removing `.all`